### ISSUE TITLE
Prerequisites for getting-started are not clearly defined

### ISSUE THREAD

**Issue by Shawn1874** [docker/docker.github.io#14987]
At File: [get-started/02_our_app.md](https://docs.docker.com/get-started/02_our_app/) it was not clear to me what yarn is.  Evidently I can install it via npm that comes with my node installation.  People that aren't familiar with javascript will not know what yarn is or how to install it, and I see nothing on the page that clearly states the pre-requisite software apps that must be installed.

**Comment by Shawn1874** [comment]
I can't reproduce it on my home computer now, even after uninstalling yarn which is odd.  I might have made a bad assumption.  I saw this line in the docker file which appears to require yarn ( RUN yarn install --production ).  What does that line do?  Is that command simply causing yarn to be installed within the container itself?  On my work computer I first had to fix https certificate issues with NPS to get farther.  I will need to try this again on my work computer on Monday to get you the error message.  Perhaps it is something unrelated to yarn and another issue with something blocked by my corporate firewall.  If I can't reproduce it by Monday evening I'll recommend we close this as unreproducible.

**Comment by Shawn1874** [comment]
Here is the error, but I agree now that it isn't yarn related.  

Microsoft Windows [Version 10.0.19043.1766]
(c) Microsoft Corporation. All rights reserved.

V:\TrustedData\Personal-Projects\Docker-TODO-List\app>docker build -t getting-started .
Sending build context to Docker daemon  4.654MB
Step 1/6 : FROM node:12-alpine
 ---> bb6d28039b8c
Step 2/6 : RUN apk add --no-cache python2 g++ make
 ---> Running in 0125c7d7c285
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
139750553811784:error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1914:
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.15/main: Permission denied
139750553811784:error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1914:
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.15/community: Permission denied
ERROR: unable to select packages:
  g++ (no such package):
    required by: world[g++]
  make (no such package):
    required by: world[make]
  python2 (no such package):
    required by: world[python2]
The command '/bin/sh -c apk add --no-cache python2 g++ make' returned a non-zero code: 3

V:\TrustedData\Personal-Projects\Docker-TODO-List\app>
